backend/game/turn_actions/player_movement.py

```diff
@@ -0,0 +1,28 @@
+from game.values.locations import Locations
+
+
+def tokyo_is_empty(attacking_player, other_players):
+    if attacking_player.location != Locations.OUTSIDE:
+        return False
+    for player in other_players:
+        if player.location != Locations.OUTSIDE:
+            return False
+    return True
+
+
+def move_to_tokyo_if_empty(attacking_player, other_players):
+    if tokyo_is_empty(attacking_player, other_players):
+        attacking_player.move_to_tokyo()
+        attacking_player.update_victory_points_by(1)
+
+
+def take_tokyo_on_kill(attacking_player, murdered_player):
```

Comment:
I laughed at murdered_player for some reason... killed_player would also work here

Reply:
I'll make it more professional. I was getting tired and snarky when I got to this. Probably defeated_player would be nicer